enterprise/views.py

```diff
@@ -270,12 +273,30 @@ def get(self, request):
             raise Http404
 
         item = 'course' if course_id else 'program'
+
+        if course_id:
+            try:
+                catalog_api_client = CourseCatalogApiServiceClient()
+            except ImproperlyConfigured:
+                raise Http404
+
+            course_run_details = catalog_api_client.get_course_and_course_run(course_id)[1]
```

Comment:
Nit: It'd probably be cleaner to just do `course_run_details, __ = catalog_api_client.get_course_and_course_run(course_id)`.

Reply:
thank you - i will make that change!

Reply:
Ended up changing to use the function 'get_course_run(course_id)' so I didn't have to use the __ notation